src/watson/registration.py

```diff
@@ -52,6 +52,12 @@ def _resolve_field(self, obj, name):
         """Resolves the content of the given model field."""
         name_parts = name.split("__", 1)
         prefix = name_parts[0]
+        # Check if the attribute exists in the database
+        try:
+            if not getattr(obj, prefix):
```

Comment:
Because if you look at the python documentation you will see that hasattr() will call getattr() to prove if the attribute exists and if you firstly call getattr on a django model it will throw the ObjectDoesNotExist Exception. So hasattr() will return False. 
Here is the source: https://docs.python.org/2/library/functions.html#hasattr